### ISSUE TITLE
Marking a test as passing when receiving an expected exception

### ISSUE THREAD

**Issue by sammcquistan** [Microsoft/EasyRepro#536]
I have a test that I am running that spits out a NullReferenceException after completing the form in a popup window. The test case works as intended and does exactly what I need it to do. Is there any way to mark a test case as passing if it receives an exception?

I tried using this format:

    [TestClass]
    public class
    {
        private readonly SecureString _username = redacted
        private readonly SecureString _password = redacted
        private readonly Uri _xrmUri = redacted

        [TestMethod]
        //next line of code is the code that i tried to mark as expected, but did not work
        [ExpectedException(typeof(NullReferenceException))]
        public void className()
        {
            using (var xrmBrowser = new Api.Browser(TestSettings.Options))
            {
                script here
             }
        }
    }

This did not work.

TL;DR - Trying to mark a test case with a false negative exception as passing. Code [ExpectedException(typeof(NulLReferenceException))] did not work. I am unable to located a similar scenario in the sample files.

Thanks!

**Comment by sammcquistan** [comment]
Solution:

Added a using directive pointing to my library of test cases. In my case, this is the following code:
using Microsoft.Dynamics365.UIAutomation.TestCases;

In the case of the sample project:
using Microsoft.Dynamics365.UIAutomation.Sample;

After adding this, I was able to use [ExpectedException{typeof(NullReferenceException))] in the same location as above. This worked successfully.
